### ISSUE TITLE
Doc: clarify forms of evaluation and supported models

### ISSUE THREAD

**Issue by ividal** [mozilla-ai/lumigator#359]
**Is your feature request related to a problem? Please describe.**
The context is [this exchange](https://github.com/mozilla-ai/lumigator/pull/357#discussion_r1840320366) around the evaluation types + models we support. 

**Describe the solution you'd like**
Docs should address:
* what offline evaluation means (and what our plans are regarding single/batch evaluation)
* what models we support (or at least interfaces) and why.

**Describe alternatives you've considered**
Past: table in the readme.md
Potentially: Allowing full flexibility (and not providing information, trusting the user to read up on their own).

**Additional context**
Since we are opinionated for evaluation, explaining what we choose and why is important.

**Comment by dpoulopoulos** [comment]
@ividal can we close this issue now that we have this [page](https://mozilla-ai.github.io/lumigator/get-started/suggested-models.html) in our docs? Do you think we need anything more?

**Comment by ividal** [comment]
Thanks for the models doc @dpoulopoulos !

Do you think we should highlight those are used for inference tasks only, whereas for evaluations other models may be used indirectly, depending on the metric chosen (e.g. bertscore)?

**Comment by dpoulopoulos** [comment]
@ividal I agree. I will make this explicit in the docs. These are the model we suggest users to try out for summarization tasks. Thus, these are the models we suggest users to evaluate on their own datasets.
